src/aws-client.js

```diff
@@ -2733,16 +2733,19 @@ require('../lib/node_loader');
 var AWS = require('../lib/core');
 var Service = AWS.Service;
 var apiLoader = AWS.apiLoader;
+var model = require('../apis/sts-2011-06-15.min.json');
 
 apiLoader.services['sts'] = {};
 AWS.STS = Service.defineService('sts', ['2011-06-15']);
 require('../lib/services/sts');
 Object.defineProperty(apiLoader.services['sts'], '2011-06-15', {
   get: function get() {
-    var model = require('../apis/sts-2011-06-15.min.json');
     model.paginators = require('../apis/sts-2011-06-15.paginators.json').pagination;
     return model;
   },
+  set: function(value) {
+    model = value;
```

Comment:
I am curious why this is getting called. Are you overriding sts somehow? Are you loading another AWS client?

Reply:
Line 20984 in **aws-client.js** `AWS.apiLoader.services['sts']['2011-06-15'] = require('../apis/sts-2011-06-15.min');` is causing the Error.

Reply:
Hi, I don't think we should be changing this as it's provided to us from the AWS SDK and it would only get wiped out should we ever update this file. We should look for some other workaround for this.